src/components/ProblemsTable.tsx

```diff
@@ -92,7 +91,37 @@ export const ProblemsTable = ({
         return context?.focusedIssues.some((issue) => bannerLevel === issue);
       });
 
-    return reports;
+    // **FIX referenced in own type annotation
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const expandableReports: any = {};
```

Comment:
where any is used as a type, can't we self-define types which would for this instance define expandableReport